### PR TITLE
vendomat inventory

### PR DESCRIPTION
**Pull request by Emisse**
vendomat inventory
idk how we havent had one this whole time

🆑 Emisse
- add: Vendomat has stuff in it now

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
restock box?

**Comment by Emisse** [comment]
ur asking a lot  out of me

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/vendomat.yml [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+  id: VendomatInventory
+  startingInventory:
+    RemoteSignaller: 1
+    Wirecutter: 1
+    FlashlightLantern: 2
+    PowerCellSmallPrinted: 3
+    MicroLaserStockPart: 3
+    MatterBinStockPart: 3
+    CapacitorStockPart: 3
+    MicroManipulatorStockPart: 3
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/vendomat.yml:11]: are they not good enough to get scanning modules, too?

**Comment by AJCM-git** [comment]
@Emisse you coming back to this? (also the restock box is kinda necessary for this to be a complete implementation)

**Comment by Emisse** [comment]
no i dont care enough to add restock box to this someone else can get to it eventually

**Pull request closed without merging**